src/SDPoptimize.jl

```diff
@@ -0,0 +1,466 @@
+#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+#  Stochastic dynamic programming algorithm
+#
+#############################################################################
+
+using ProgressMeter
+
+"""
+Convert the state and control tuples (stored as arrays) of the problem into integers and vice versa
+
+"""
+function index_from_variable( variable::Array,
```

Comment:
and we should think about using a decorator for these two functions (or use metaprogramming instead)